{"name": "click cart icon", "status": "broken", "statusDetails": {"message": "Exception: Element by \"id\" with the value \"cart\" cannot be found.", "trace": "E   selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\n    from unknown error: web view not found\n      (Session info: chrome=101.0.4951.54)\n    Stacktrace:\n    Backtrace:\n    \tOrdinal0 [0x002AB8F3+2406643]\n    \tOrdinal0 [0x0023AF31+1945393]\n    \tOrdinal0 [0x0012C748+837448]\n    \tOrdinal0 [0x001142B0+737968]\n    \tOrdinal0 [0x00176C09+1141769]\n    \tOrdinal0 [0x00183E22+1195554]\n    \tOrdinal0 [0x00174096+1130646]\n    \tOrdinal0 [0x0014E636+976438]\n    \tOrdinal0 [0x0014F546+980294]\n    \tGetHandleVerifier [0x00519612+2498066]\n    \tGetHandleVerifier [0x0050C920+2445600]\n    \tGetHandleVerifier [0x00344F2A+579370]\n    \tGetHandleVerifier [0x00343D36+574774]\n    \tOrdinal0 [0x00241C0B+1973259]\n    \tOrdinal0 [0x00246688+1992328]\n    \tOrdinal0 [0x00246775+1992565]\n    \tOrdinal0 [0x0024F8D1+2029777]\n    \tBaseThreadInitThunk [0x763A6739+25]\n    \tRtlGetFullPathName_UEx [0x77278E7F+1215]\n    \tRtlGetFullPathName_UEx [0x77278E4D+1165]\n    \t(No symbol) [0x00000000]\n\nDuring handling of the above exception, another exception occurred:\nE   Exception: Element by \"id\" with the value \"cart\" cannot be found."}, "description": "go to site and click icon", "steps": [{"name": "click cart icon", "status": "broken", "statusDetails": {"message": "Exception: Element by \"id\" with the value \"cart\" cannot be found.\n", "trace": "  File \"C:\\Users\\pc\\PycharmProjects\\pythonProject1\\SmartBearProject1\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\pc\\PycharmProjects\\pythonProject1\\SmartBearProject1\\Pages\\HomePage.py\", line 12, in click_cart_icon\n    cart_page = BasePage.find_element(self,Locators.cart_icon_id)\n  File \"C:\\Users\\pc\\PycharmProjects\\pythonProject1\\SmartBearProject1\\Pages\\BasePage.py\", line 16, in find_element\n    raise Exception(\n"}, "start": 1652685687518, "stop": 1652685687520}], "start": 1652685687517, "stop": 1652685687527, "uuid": "0ee78d4c-221e-4a9b-be1c-47b11023953b", "historyId": "11d1795ffaa4565162daa106512b311a", "testCaseId": "2ceb92be4affa658ba4e4b8a170aa726", "fullName": "Tests.Test_SmartBear.Test_SmartBear#test_click_cart_icon", "labels": [{"name": "story", "value": "FinalProject"}, {"name": "feature", "value": "SmartBear Test Automation Add Item To Cart"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "Test_SmartBear"}, {"name": "subSuite", "value": "Test_SmartBear"}, {"name": "host", "value": "DESKTOP-8MCCQ5O"}, {"name": "thread", "value": "8464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.Test_SmartBear"}]}